Packages/io.chainsafe.web3-unity/Runtime/Scripts/CountlyAnalytics.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Web3.Analytics;
+using Plugins.CountlySDK;
+using Plugins.CountlySDK.Models;
+using UnityEngine;
+
+public class CountlyAnalytics : IAnalyticsClient
+{
+    private const string AppKey = "9fad27b7383f5151a316505ca3245287fc44ba50";
```

Comment:
I think we should hide it somewhere if we don't want other people using the App Key. Environment Variables is a good place probably. @RyRy79261 or @robGG1997 may know better.

Reply:
Maybe obfuscator the class?

Reply:
So the only way I think we could hid this is be using a server/intermediary

Reply:
Yeah asked Peter, he said nothing we can do about it.